options.go

```diff
@@ -3,11 +3,12 @@ package rediswatcher
 import "github.com/garyburd/redigo/redis"
 
 type WatcherOptions struct {
-	Channel    	string
-	PubConn 	redis.Conn
-	SubConn		redis.Conn
-	Password   	string
-	Protocol   	string
+	Channel   string
+	PubConn   redis.Conn
+	SubConn   redis.Conn
+	Password  string
+	Protocol  string
+	Committed chan struct{} // for transaction, we should do: Committed <- struct{}{} after transaction is committed.
```

Comment:
I'm not sure about exposing a channel via the API. This seems like something that should be coordinated before Casbin invokes `Update()`.